x-pack/plugins/apm/e2e/cypress/integration/rum_dashboard.feature

```diff
@@ -5,3 +5,19 @@ Feature: RUM Dashboard
     When the user inspects the real user monitoring tab
     Then should redirect to rum dashboard
       And should have correct client metrics
+
+  Scenario: Page load distribution
+    Given a user browses the APM UI application for RUM Data
+    When the user inspects the real user monitoring tab
+    Then should redirect to rum dashboard
+      And should display percentile for page load chart
+      And should display tooltip on hover
+      And should display chart legend
```

Comment:
Having multiple `Ands` gives the sense of _scripting_, which is something to be avoided when writing BDD features. I'd suggest rethinking about this three clauses into a high-level one. In the JS code you can combine the three of them into the single step.